### PR TITLE
Removed Spring from the dependencies

### PR DESCRIPTION
**Pull request by mowczare**

### CODE REVIEW

**Comment by mowczare** [comment]
closes https://github.com/UdashFramework/udash-core/issues/376

File: src/main/g8/backend/src/main/scala/$package$/backend/Launcher.scala [hunk 1]
```diff
@@ -12,8 +13,15 @@ object Launcher extends CrossLogging {
...
-    val ctx = SpringContext.createApplicationContext("beans.conf")
-    val server = ctx.getBean(classOf[ApplicationServer])
+    val config: Config = ConfigFactory.load()
+    val rpcClientsService: RpcClientsService = new RpcClientsService(RpcClientsService.defaultSendToClientFactory)
+    val authService: AuthService = new AuthService(config.getStringList("auth.users"))
+    val chatService: ChatService = new ChatService(rpcClientsService)
+    val server = new ApplicationServer(
+      config.getInt("server.port"),
+      config.getString("server.statics"),
+      new DomainServices()(authService, chatService, rpcClientsService)
```
Comment (ddworak) [inline src/main/g8/backend/src/main/scala/$package$/backend/Launcher.scala]: I think implicit wiring in this case is a pretty good fit. These services are singletons anyway.

File: src/main/g8/project/Dependencies.scala [hunk 2]
```diff
@@ -85,12 +85,7 @@ object Dependencies {
     "org.eclipse.jetty" % "jetty-rewrite" % jettyVersion,
     "org.eclipse.jetty.websocket" % "websocket-server" % jettyVersion,
 
-    "org.springframework" % "spring-core" % springVersion,
-    "org.springframework" % "spring-beans" % springVersion,
-    "org.springframework" % "spring-context-support" % springVersion,
-
-    // support for HOCON beans configuration
-    "com.avsystem.commons" %% "commons-spring" % avsystemCommonsVersion,
```
Comment (ddworak) [inline src/main/g8/project/Dependencies.scala:93]: commons version variable is now unused

**Comment by mowczare** [comment]
@ddworak please re-review this please

**Review by ddworak — APPROVED** [review]

**Pull request merged**